.github/workflows/build-container-images.yaml

```diff
@@ -7,18 +7,22 @@ on:
     paths:
       - 'argo-workflows/generic/containers/*'
       - 'argo-workflows/ironic-nautobot-sync/containers/*'
+      - 'argo-workflows/obm-utils/containers/*'
```

Comment:
It's not related only to your PR/feature, but do you think we should also trigger this on any change under the `code/` folder too?